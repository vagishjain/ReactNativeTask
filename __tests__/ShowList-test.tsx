import React from 'react';
import renderer from 'react-test-renderer';
import ShowListScreen from '../src/Screens/ShowList/ShowList.screen';
import ReduxProvider from "../src/Components/Provider.wrapper";

test('Test ShowListScreen', () => {
    const tree = renderer.create(<ReduxProvider><ShowListScreen /></ReduxProvider>).toJSON();
  expect(tree).toMatchSnapshot();
});
