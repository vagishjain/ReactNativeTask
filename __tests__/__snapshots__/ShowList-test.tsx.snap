// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ShowListScreen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
    }
  }
>
  <RCTScrollView
    ItemSeparatorComponent={[Function]}
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onRefresh={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    progressViewOffset={60}
    refreshControl={
      <RefreshControlMock
        onRefresh={[Function]}
        progressViewOffset={60}
        refreshing={false}
      />
    }
    refreshing={false}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <RCTRefreshControl />
    <View />
  </RCTScrollView>
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
    }
  }
>
  <RCTScrollView
    ItemSeparatorComponent={[Function]}
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onRefresh={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    progressViewOffset={60}
    refreshControl={
      <RefreshControlMock
        onRefresh={[Function]}
        progressViewOffset={60}
        refreshing={false}
      />
    }
    refreshing={false}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <RCTRefreshControl />
    <View />
  </RCTScrollView>
</View>
`;
